commands/report_generate.js

```diff
@@ -0,0 +1,83 @@
+const request = require("request")
+const constants = require("./utils/constants.js")
+const process = require("process")
+
+
+function generate_report(args) {
+
+    return new Promise(function (resolve, reject) {
+        var username = ""
+        var access_key = ""
+        //Check for username
+        if ("user" in args) {
```

Comment:
Please use `hasOwnProperty()` because `hasOwnProperty()` searches only within the own properties of the object.